pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -29,6 +29,13 @@ let
           (stdenv.hostPlatform.isPower && stdenv.hostPlatform.is64bit) ||
           (stdenv.hostPlatform.isMips && stdenv.hostPlatform.is64bit));
 
+  forceRust = features.rust or false;
+  kernelSupportsRust = lib.versionAtLeast version "6.7";
+
+  # Currently not enabling Rust by default, as upstream requires rustc 1.81
+  defaultRust = false;
+  withRust = (forceRust || defaultRust) && kernelSupportsRust;
```

Comment:
Shouldn't we complain loudly if Rust is explicitly requested, but your kernel is too old?

Or is this done elsewhere already?

Reply:
It's not, but maybe we should, yeah.

Reply:
OK won't block on that since it was the case already. Would file a follow-up later (or tomorrow).